ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageView.java

```diff
@@ -217,21 +277,31 @@ public void maybeUpdateView() {
         .setProgressiveRenderingEnabled(mProgressiveRenderingEnabled)
         .build();
 
-    DraweeController draweeController = mDraweeControllerBuilder
+    mDraweeControllerBuilder
```

Comment:
nit: separate call to `.reset()` with a comment that the same builder is going to be reused, then continue with all the remaining calls